test/image/mocks/gl3d_traces-with-opacity.json

```diff
@@ -0,0 +1,173 @@
+    "type": "surface",
+    "name": "surface",
+    "scene": "scene1"
+  }, {
+    "x": [0, 1, 0, 1, 0, 1, 0, 1],
+    "y": [0, 0, 1, 1, 0, 0, 1, 1],
+    "z": [0, 0, 0, 0, 1, 1, 1, 1],
+    "i": [0, 3, 4, 7, 0, 6, 1, 7, 0, 5, 2, 7],
+    "j": [1, 2, 5, 6, 2, 4, 3, 5, 4, 1, 6, 3],
+    "k": [3, 0, 7, 4, 6, 0, 7, 1, 5, 0, 7, 2],
+    "flatshading": true,
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 1,
+    "type": "mesh3d",
+    "name": "mesh3d",
+    "scene": "scene2"
+  }, {
+    "x": [-1, 0, 1],
+    "y": [-1, 0, 1],
+    "z": [
+      [0, 1, 0],
+      [1, 0, 1],
+      [0, 1, 0]
+    ],
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 0.5,
+    "showscale": false,
+    "type": "surface",
+    "name": "surface",
+    "scene": "scene3"
+  }, {
+    "x": [0, 1, 0, 1, 0, 1, 0, 1],
+    "y": [0, 0, 1, 1, 0, 0, 1, 1],
+    "z": [0, 0, 0, 0, 1, 1, 1, 1],
+    "i": [0, 3, 4, 7, 0, 6, 1, 7, 0, 5, 2, 7],
+    "j": [1, 2, 5, 6, 2, 4, 3, 5, 4, 1, 6, 3],
+    "k": [3, 0, 7, 4, 6, 0, 7, 1, 5, 0, 7, 2],
+    "flatshading": true,
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 0.5,
+    "type": "mesh3d",
+    "name": "mesh3d",
+    "scene": "scene4"
+  }, {
+    "x": [0, 0, 1, 1, 0, 0, 1, 1],
+    "y": [0, 1, 1, 0, 0, 1, 1, 0],
+    "z": [0, 0, 0, 0, 1, 1, 1, 1],
+    "u": [0, 0, 1, 1, 0, 0, 1, 1],
+    "v": [0, 1, 1, 0, 0, 1, 1, 0],
+    "w": [0, 0, 0, 0, 1, 1, 1, 1],
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 0.5,
+    "showscale": false,
+    "type": "cone",
+    "name": "cone",
+    "scene": "scene5"
+  }, {
+    "x": [0, 0, 1, 1, 0, 0, 1, 1],
+    "y": [0, 1, 1, 0, 0, 1, 1, 0],
+    "z": [0, 0, 0, 0, 1, 1, 1, 1],
+    "u": [0, 0, 1, 1, 0, 0, 1, 1],
+    "v": [0, 1, 1, 0, 0, 1, 1, 0],
+    "w": [0, 0, 0, 0, 1, 1, 1, 1],
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 0.5,
+    "showscale": false,
+    "type": "streamtube",
+    "name": "streamtube",
+    "scene": "scene6"
+  }, {
+    "x": [0, 0, 0, 0, 1, 1, 1, 1],
+    "y": [0, 0, 1, 1, 0, 0, 1, 1],
+    "z": [0, 1, 0, 1, 0, 1, 0, 1],
+    "value": [0, 0, 0, 1, 1, 0, 0, 0],
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacity": 0.5,
+    "showscale": false,
+    "type": "isosurface",
+    "name": "isosurface",
+    "scene": "scene7"
+  }, {
+    "x": [0, 0, 0, 0, 1, 1, 1, 1],
+    "y": [0, 0, 1, 1, 0, 0, 1, 1],
+    "z": [0, 1, 0, 1, 0, 1, 0, 1],
+    "value": [0, 0, 0, 1, 1, 0, 0, 0],
+    "lightposition": { "x": 1000, "y": 1000, "z": 0 },
+    "opacityscale": "max",
+    "opacity": 0.5,
+    "showscale": false,
+    "type": "volume",
+    "name": "volume",
+    "scene": "scene8"
+  }],
+  "layout": {
+    "paper_bgcolor": "rgba(0,0,255, 0.5)",
+    "margin": {
+      "t": 0,
+      "b": 0,
+      "l": 0,
+      "r": 0
+    },
+    "width": 1000,
+    "height": 500,
+    "scene1": {
+      "domain": {
+        "x": [0, 0.25],
+        "y": [0, 0.25]
```

Comment:
```suggestion
        "y": [0, 0.5]
```

Reply:
Good eye! Done in https://github.com/plotly/plotly.js/pull/4344/commits/0dd87488852d7950fa8b9757d188dc8b3506ba82.
I also think we may want to close https://github.com/plotly/plotly.js/issues/2477.